### DISCUSSION TITLE
Welcome to foyle Discussions!

### DISCUSSION THREAD

**Discussion by jlewi** [jlewi/foyle#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by menithya** [comment]
how do you install in mac, I have cloned the repo, unable to run 
`zsh: command not found: foyle`

**Comment by jlewi** [comment]
@menithya You should follow the [Getting Started Page](https://foyle.io/docs/getting-started/). You can download the binary from the [releases page](https://github.com/jlewi/foyle/releases)

**Comment by varunsh17** [comment]
Hi @jlewi , great meeting you! 
I would like to open source in Foyle. It would be great if I can have a slack channel or something sort of for regular discussion?

**Reply by jlewi** [reply]
Welcome.

Lets use GitHub issues and discussions and see how far that gets us.
